resources/views/layouts/sidebars/quick-links.blade.php

```diff
@@ -13,35 +13,37 @@
 
         <ul class="sidebar-submenu">
             @foreach ($currentCampaign->menuLinks()->with(['target'])->ordered()->get() as $menuLink)
-                <?php /** @var \App\Models\MenuLink $menuLink */ ?>
-                @if ($menuLink->dashboard && $currentCampaign->boosted() && $menuLink->isValidDashboard())
-                    <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }}">
-                        <a href="{{ $menuLink->getRoute() }}">
-                            <i class="{{ $menuLink->icon() }}"></i>
-                            {{ $menuLink->name }}
-                        </a>
-                    </li>
-                @elseif ($menuLink->target)
-                    <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }}">
-                        <a href="{{ $menuLink->getRoute() }}">
-                            <i class="{{ $menuLink->icon() }}"></i>
-                            {{ $menuLink->name }}
-                        </a>
-                    </li>
-                @elseif ($menuLink->type)
-                    <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }}">
-                        <a href="{{ $menuLink->getRoute() }}">
-                            <i class="{{ $menuLink->icon() }}"></i>
-                            {{ $menuLink->name }}
-                        </a>
-                    </li>
-                @elseif ($menuLink->isRandom())
-                    <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }}">
-                        <a href="{{ route('menu_links.random', $menuLink) }}">
-                            <i class="{{ $menuLink->icon() }}"></i>
-                            {{ $menuLink->name }}
-                        </a>
-                    </li>
+                @if (!$menuLink->parent || $menuLink->parent && !$currentCampaign->boosted())
```

Comment:
Could use some extra () to make it clear just by reading which conditions are grouped together